### ISSUE TITLE
[BUG] Outbound Transactions vanish

### ISSUE THREAD

**Issue by sirdeggen** [BuxOrg/bux-server#416]
**TL;DR**
Check if your issue answers those question:

- [x]  Does my issue clearly describe what exactly is not working properly?
- [x]  Does my issue clearly describe what are my expectations and what is the current output?
- [x]  Did I attach logs/screenshots/errors which occured?
- [x]  Did I provide information how I run the code/library/application and on which platform/browser?

**Describe the bug**
Outbound transactions appear only for a few seconds, when the app refreshes they disappear never to come back.
The receiving wallets _do_ see the transactions and they _are_ on chain. Problem is intermittent, some txs are showing, just not all.

**To Reproduce**
1. Set up an account at bux-wallet.domain.tld
2. Send money to if from handcash
3. Send a transaction back to handcash

**Expected behavior**
Sent transactions should continue to be displayed as part of transactions history.

**Logs**
```bash
// bux-server 
2024-01-19 20:34:55.956 DBG try add write lock action-record-transaction-outgoing-fcafbb85be37b9c460ffd27df0a935a338989ca6ac9b37974b659c6fd515753e application=bux-server ecs.version=1.6.0 service=bux
2024-01-19 20:34:55.959 DBG added write lock action-record-transaction-outgoing-fcafbb85be37b9c460ffd27df0a935a338989ca6ac9b37974b659c6fd515753e application=bux-server ecs.version=1.6.0 service=bux
2024-01-19 20:34:55.960 INF start recording transaction application=bux-server ecs.version=1.6.0 service=bux txID=fcafbb85be37b9c460ffd27df0a935a338989ca6ac9b37974b659c6fd515753e
2024-01-19 20:34:55.962 INF executing query application=bux-server duration=0.859ms ecs.version=1.6.0 file=/go/pkg/mod/github.com/!bux!org/bux@v0.11.0/logging/adapters.go:63 rows=1 service=bux sql="SELECT * FROM \"xapi_draft_transactions\" WHERE xpub_id = 'ce9d50bd414b35eb415b69cbc88aad4e4917c68f4b30433e1f0202d62d5eac24' AND id = 'e6f4ae1794647d38ab9b544840eb6c6486a0539398f0835c5b2e90ae273118da'" subservice=datastore
2024-01-19 20:34:55.964 INF executing query application=bux-server duration=0.444ms ecs.version=1.6.0 file=/go/pkg/mod/github.com/!bux!org/bux@v0.11.0/logging/adapters.go:63 rows=1 service=bux sql="SELECT * FROM \"xapi_utxos\" WHERE transaction_id = 'b973aa86b4843f5e1704a38d07b6b241c92d01592412eee530cb56a6fcd0a0e0' AND output_index = 0" subservice=datastore
2024-01-19 20:34:55.964 DBG unlock action-record-transaction-outgoing-fcafbb85be37b9c460ffd27df0a935a338989ca6ac9b37974b659c6fd515753e application=bux-server ecs.version=1.6.0 service=bux
2024-01-19 20:34:55.968 DBG 422 | 14.438418ms | 10.0.0.105:45450 | POST | /v1/transaction/record  application=bux-server ecs.version=1.6.0 elapsed=14.438418ms logger=http-request remote=10.0.0.105:45450 remote_address=10.0.0.105:45450 service=bux-server status=422 url=/v1/transaction/record
2024/01/19 20:34:55 http: panic serving 10.0.0.105:45450: runtime error: invalid memory address or nil pointer dereference
goroutine 124950 [running]:
net/http.(*conn).serve.func1()
	/usr/local/go/src/net/http/server.go:1868 +0xb9
panic({0x1758a40?, 0x28c85d0?})
	/usr/local/go/src/runtime/panic.go:920 +0x270
github.com/BuxOrg/bux.unReserveUtxos({0x1d656d8, 0xc0018914a0}, {0xc001230240, 0x40}, {0xc0012301c0, 0x40}, {0x0, 0x0, 0x1d52ae0?})
	/go/pkg/mod/github.com/!bux!org/bux@v0.11.0/model_utxos.go:123 +0x1d5
github.com/BuxOrg/bux.(*Client).UnReserveUtxos(0x18?, {0x1d656d8?, 0xc0018914a0?}, {0xc001230240, 0x40}, {0xc0012301c0, 0x40})
	/go/pkg/mod/github.com/!bux!org/bux@v0.11.0/action_utxo.go:139 +0x6d
github.com/BuxOrg/bux-server/actions/utxos.(*Action).unreserve(0xc00103cf60, {0x1d62a38, 0xc000fb1800}, 0xc001242100, {0xc000ff3e90?, 0xc001ac7758?, 0x646eb2?})
	/go/src/github.com/BuxOrg/bux-server/actions/utxos/unreserve.go:24 +0xd5
github.com/BuxOrg/bux-server/actions.(*Action).RequireAuthentication-fm.(*Action).RequireAuthentication.func1({0x1d62a38, 0xc000fb1800}, 0x0?, {0x0, 0x0, 0x0})
	/go/src/github.com/BuxOrg/bux-server/actions/middleware.go:40 +0x184
github.com/BuxOrg/bux-server/actions/utxos.RegisterRoutes.(*Action).Request.Request.func4({0x1d62228?, 0xc001260ff0}, 0xc00111dd00, {0x0, 0x0, 0x0})
	/go/src/github.com/BuxOrg/bux-server/actions/middleware.go:124 +0x29f
github.com/BuxOrg/bux-server/actions/utxos.RegisterRoutes.(*Action).Request.Request.MakeHTTPRouterParsedReq.func8({0x1d62228, 0xc001260ff0}, 0xc001174700?, {0x0?, 0x0, 0x0})
	/go/pkg/mod/github.com/mrz1836/go-parameters@v0.4.1/parameters.go:832 +0x2db
github.com/newrelic/go-agent/v3/integrations/nrhttprouter.(*Router).handle.func1({0x1d62588, 0xc001174700}, 0xc001204a50?, {0x0, 0x0, 0x0})
	/go/pkg/mod/github.com/newrelic/go-agent/v3/integrations/nrhttprouter@v1.0.2/nrhttprouter.go:83 +0x189
github.com/julienschmidt/httprouter.(*Router).ServeHTTP(0xc000d91b60, {0x1d62588, 0xc001174700}, 0xc00111db00)
	/go/pkg/mod/github.com/julienschmidt/httprouter@v1.3.0/router.go:387 +0x7eb
github.com/newrelic/go-agent/v3/integrations/nrhttprouter.(*Router).ServeHTTP(0xc00103c710?, {0x1d62588?, 0xc001174700?}, 0xc00111db00?)
	/go/pkg/mod/github.com/newrelic/go-agent/v3/integrations/nrhttprouter@v1.0.2/nrhttprouter.go:155 +0x178
net/http.serverHandler.ServeHTTP({0x1d5d658?}, {0x1d62588?, 0xc001174700?}, 0x6?)
	/usr/local/go/src/net/http/server.go:2938 +0x8e
net/http.(*conn).serve(0xc0012a5710, {0x1d656d8, 0xc001036db0})
	/usr/local/go/src/net/http/server.go:2009 +0x5f4
created by net/http.(*Server).Serve in goroutine 1
	/usr/local/go/src/net/http/server.go:3086 +0x5cb
2024/01/19 20:34:57 http: panic serving 10.0.0.105:54154: runtime error: invalid memory address or nil pointer dereference
goroutine 124332 [running]:
net/http.(*conn).serve.func1()
	/usr/local/go/src/net/http/server.go:1868 +0xb9
panic({0x1758a40?, 0x28c85d0?})
	/usr/local/go/src/runtime/panic.go:920 +0x270
github.com/BuxOrg/bux.unReserveUtxos({0x1d656d8, 0xc00183bf20}, {0xc00119f140, 0x40}, {0xc00119f0c0, 0x40}, {0x0, 0x0, 0x1d52ae0?})
	/go/pkg/mod/github.com/!bux!org/bux@v0.11.0/model_utxos.go:123 +0x1d5
github.com/BuxOrg/bux.(*Client).UnReserveUtxos(0x18?, {0x1d656d8?, 0xc00183bf20?}, {0xc00119f140, 0x40}, {0xc00119f0c0, 0x40})
	/go/pkg/mod/github.com/!bux!org/bux@v0.11.0/action_utxo.go:139 +0x6d
github.com/BuxOrg/bux-server/actions/utxos.(*Action).unreserve(0xc00103cf60, {0x1d62a38, 0xc0015bde00}, 0xc001185400, {0xc0013cedf8?, 0xc001527758?, 0x646eb2?})
	/go/src/github.com/BuxOrg/bux-server/actions/utxos/unreserve.go:24 +0xd5
github.com/BuxOrg/bux-server/actions.(*Action).RequireAuthentication-fm.(*Action).RequireAuthentication.func1({0x1d62a38, 0xc0015bde00}, 0x0?, {0x0, 0x0, 0x0})
	/go/src/github.com/BuxOrg/bux-server/actions/middleware.go:40 +0x184
github.com/BuxOrg/bux-server/actions/utxos.RegisterRoutes.(*Action).Request.Request.func4({0x1d62228?, 0xc00104f9f0}, 0xc001185000, {0x0, 0x0, 0x0})
	/go/src/github.com/BuxOrg/bux-server/actions/middleware.go:124 +0x29f
github.com/BuxOrg/bux-server/actions/utxos.RegisterRoutes.(*Action).Request.Request.MakeHTTPRouterParsedReq.func8({0x1d62228, 0xc00104f9f0}, 0xc0010f28c0?, {0x0?, 0x0, 0x0})
	/go/pkg/mod/github.com/mrz1836/go-parameters@v0.4.1/parameters.go:832 +0x2db
github.com/newrelic/go-agent/v3/integrations/nrhttprouter.(*Router).handle.func1({0x1d62588, 0xc0010f28c0}, 0xc00055a150?, {0x0, 0x0, 0x0})
	/go/pkg/mod/github.com/newrelic/go-agent/v3/integrations/nrhttprouter@v1.0.2/nrhttprouter.go:83 +0x189
github.com/julienschmidt/httprouter.(*Router).ServeHTTP(0xc000d91b60, {0x1d62588, 0xc0010f28c0}, 0xc001184e00)
	/go/pkg/mod/github.com/julienschmidt/httprouter@v1.3.0/router.go:387 +0x7eb
github.com/newrelic/go-agent/v3/integrations/nrhttprouter.(*Router).ServeHTTP(0xc00103c710?, {0x1d62588?, 0xc0010f28c0?}, 0xc001184e00?)
	/go/pkg/mod/github.com/newrelic/go-agent/v3/integrations/nrhttprouter@v1.0.2/nrhttprouter.go:155 +0x178
net/http.serverHandler.ServeHTTP({0x1d5d658?}, {0x1d62588?, 0xc0010f28c0?}, 0x6?)
	/usr/local/go/src/net/http/server.go:2938 +0x8e
net/http.(*conn).serve(0xc0012a55f0, {0x1d656d8, 0xc001036db0})
	/usr/local/go/src/net/http/server.go:2009 +0x5f4
created by net/http.(*Server).Serve in goroutine 1
	/usr/local/go/src/net/http/server.go:3086 +0x5cb
2024/01/19 20:34:59 http: panic serving 10.0.0.105:50420: runtime error: invalid memory address or nil pointer dereference
goroutine 125112 [running]:
net/http.(*conn).serve.func1()
	/usr/local/go/src/net/http/server.go:1868 +0xb9
panic({0x1758a40?, 0x28c85d0?})
	/usr/local/go/src/runtime/panic.go:920 +0x270
github.com/BuxOrg/bux.unReserveUtxos({0x1d656d8, 0xc0018b8390}, {0xc0011c0b00, 0x40}, {0xc0011c0a80, 0x40}, {0x0, 0x0, 0x1d52ae0?})
	/go/pkg/mod/github.com/!bux!org/bux@v0.11.0/model_utxos.go:123 +0x1d5
github.com/BuxOrg/bux.(*Client).UnReserveUtxos(0x18?, {0x1d656d8?, 0xc0018b8390?}, {0xc0011c0b00, 0x40}, {0xc0011c0a80, 0x40})
	/go/pkg/mod/github.com/!bux!org/bux@v0.11.0/action_utxo.go:139 +0x6d
github.com/BuxOrg/bux-server/actions/utxos.(*Action).unreserve(0xc00103cf60, {0x1d62a38, 0xc0015bdec0}, 0xc001185b00, {0xc0013cf020?, 0xc001adf758?, 0x646eb2?})
	/go/src/github.com/BuxOrg/bux-server/actions/utxos/unreserve.go:24 +0xd5
github.com/BuxOrg/bux-server/actions.(*Action).RequireAuthentication-fm.(*Action).RequireAuthentication.func1({0x1d62a38, 0xc0015bdec0}, 0x0?, {0x0, 0x0, 0x0})
	/go/src/github.com/BuxOrg/bux-server/actions/middleware.go:40 +0x184
github.com/BuxOrg/bux-server/actions/utxos.RegisterRoutes.(*Action).Request.Request.func4({0x1d62228?, 0xc00104fb80}, 0xc001185700, {0x0, 0x0, 0x0})
	/go/src/github.com/BuxOrg/bux-server/actions/middleware.go:124 +0x29f
github.com/BuxOrg/bux-server/actions/utxos.RegisterRoutes.(*Action).Request.Request.MakeHTTPRouterParsedReq.func8({0x1d62228, 0xc00104fb80}, 0xc0010f2a80?, {0x0?, 0x0, 0x0})
	/go/pkg/mod/github.com/mrz1836/go-parameters@v0.4.1/parameters.go:832 +0x2db
github.com/newrelic/go-agent/v3/integrations/nrhttprouter.(*Router).handle.func1({0x1d62588, 0xc0010f2a80}, 0xc00055b050?, {0x0, 0x0, 0x0})
	/go/pkg/mod/github.com/newrelic/go-agent/v3/integrations/nrhttprouter@v1.0.2/nrhttprouter.go:83 +0x189
github.com/julienschmidt/httprouter.(*Router).ServeHTTP(0xc000d91b60, {0x1d62588, 0xc0010f2a80}, 0xc001185500)
	/go/pkg/mod/github.com/julienschmidt/httprouter@v1.3.0/router.go:387 +0x7eb
github.com/newrelic/go-agent/v3/integrations/nrhttprouter.(*Router).ServeHTTP(0xc00103c710?, {0x1d62588?, 0xc0010f2a80?}, 0xc001185500?)
	/go/pkg/mod/github.com/newrelic/go-agent/v3/integrations/nrhttprouter@v1.0.2/nrhttprouter.go:155 +0x178
net/http.serverHandler.ServeHTTP({0x1d5d658?}, {0x1d62588?, 0xc0010f2a80?}, 0x6?)
	/usr/local/go/src/net/http/server.go:2938 +0x8e
net/http.(*conn).serve(0xc0011953b0, {0x1d656d8, 0xc001036db0})
	/usr/local/go/src/net/http/server.go:2009 +0x5f4
created by net/http.(*Server).Serve in goroutine 1
	/usr/local/go/src/net/http/server.go:3086 +0x5cb
```

**Used environment (please complete the following information):**
AWS deployment from CFT EKS
MacOS chrome latest
